src/ducks/apps/components/Sections/Sections.jsx

```diff
@@ -0,0 +1,167 @@
+      filteredApps.filter(a => a.type === APP_TYPE.KONNECTOR)
+    )
+    const webAppGroups = catUtils.groupApps(
+      filteredApps.filter(a => a.type === APP_TYPE.WEBAPP)
+    )
+    const webAppsCategories = Object.keys(webAppGroups)
+      .map(cat => catUtils.addLabel({ value: cat }, t))
+      .sort(catUtils.sorter)
+    const konnectorsCategories = Object.keys(konnectorGroups)
+      .map(cat => catUtils.addLabel({ value: cat }, t))
+      .sort(catUtils.sorter)
+
+    const dropdownDisplayed = hasNav && (isMobile || isTablet)
+    const rawSelectOptions = catUtils.generateOptionsFromApps(apps, true)
+    const selectOptions = rawSelectOptions.map(option =>
+      catUtils.addLabel(option, t)
+    )
+    const optionMatcher = makeOptionMatcherFromSearch(this.state.searchParams)
+    const defaultFilterValue = selectOptions.find(optionMatcher)
+
+    return (
+      <div className={`sto-sections${dropdownDisplayed ? '' : ' u-mt-half'}`}>
+        {dropdownDisplayed && (
+          <DropdownFilter
+            defaultValue={defaultFilterValue}
+            options={selectOptions}
+            onChange={this.handleSearchOptionChange}
+          />
+        )}
+        {!isMobile && !!webAppsCategories.length && (
+          <h1 className="sto-sections-title u-title-h1">
+            {t('sections.applications')}
+          </h1>
+        )}
+        <div className="sto-sections-section">
+          {!!webAppsCategories.length && (
+            <div>
+              {webAppsCategories.map(cat => {
+                return (
+                  <AppsSection
+                    key={cat.value}
+                    appsList={webAppGroups[cat.value]}
+                    subtitle={
+                      <h2 className="sto-sections-subtitle u-title-h2">
+                        {cat.label}
+                      </h2>
+                    }
+                    onAppClick={onAppClick}
+                  />
+                )
+              })}
+            </div>
+          )}
+          {!!konnectorsCategories.length && (
+            <div>
+              <h2 className="sto-sections-subtitle u-title-h2">
+                {t('sections.konnectors')}
+              </h2>
+              {konnectorsCategories.map(cat => {
+                return (
+                  <AppsSection
+                    key={cat.value}
+                    appsList={konnectorGroups[cat.value]}
+                    subtitle={
+                      <h3 className="sto-sections-subtitle-secondary">
+                        {cat.label}
+                      </h3>
+                    }
+                    onAppClick={onAppClick}
+                  />
+                )
+              })}
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }
+}
+
+Sections.propTypes = {
+  t: PropTypes.func.isRequired,
+  apps: PropTypes.array.isRequired,
+  error: PropTypes.object,
+  onAppClick: PropTypes.func.isRequired
```

Comment:
hasNav: PropTypes.bool.isRequired

Reply:
Well spotted, I did not put the required since hasNav is in the defaultProps ;)